<ui version="4.0" >
 <class>ListAddDialog</class>
 <widget class="QDialog" name="ListAddDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>415</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Add Blocklist</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images/images.qrc" >:/images/mobloquer.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_ListLocationEdit" >
          <property name="toolTip" >
           <string>The location of the blocklist. Can be a local or a remote location.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_FileSelectButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <pointsize>8</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text" >
         <string>The location of the Blocklist. It can either be a local path or an url to a remote server. Note that all blocklists used by MoBlock must have the same format. So be careful not to add multiple Blocklists with different formats.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_NoTimeStampBox" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string>No time stamping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <pointsize>8</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text" >
         <string>This feature is deprecated and has no special use any more. Just ignore it.</string>
        </property>
        <property name="scaledContents" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_LocalBox" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <pointsize>8</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text" >
         <string>Check this is the Blocklist is stored in your Hard Drive.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ListAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ListAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
